apps/performance/37-optimize-big-list/src/app/person-list.component.ts

```diff
@@ -1,24 +1,25 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
-
 import { NgForTrackByModule } from '@angular-challenges/shared/directives';
+import { ScrollingModule } from '@angular/cdk/scrolling';
 import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import { Person } from './person.model';
 
 @Component({
   selector: 'app-person-list',
   standalone: true,
-  imports: [CommonModule, NgForTrackByModule],
+  imports: [CommonModule, NgForTrackByModule, ScrollingModule],
   template: `
-    <div class="relative h-[300px] overflow-hidden">
-      <div class="absolute inset-0 overflow-scroll">
-        <div
-          *ngFor="let person of persons; trackByProp: 'email'"
-          class="flex h-9 items-center justify-between border-b">
-          <h3>{{ person.name }}</h3>
-          <p>{{ person.email }}</p>
-        </div>
+    <cdk-virtual-scroll-viewport
+      itemSize="20"
+      class="example-viewport"
+      appendOnly>
```

Comment:
question: What is the appendOnly use for ?

Reply:
The previous items are not deleted once the list is scrolled and new items are loaded. 
Why it was used?
I observed that when scrolling back the layout was shifting because of such a large list and hence appendOnly was used.